### DISCUSSION TITLE
The Platform Support Image version (server.psi) is lower than the requested schema version

### DISCUSSION THREAD

**Discussion by georgeinva2004** [zwave-js/ZWaveJS.NET#40] [General]
Finally got some time to refactor my application to use 4.0.0.  I'm using the prebuilt PSI and am getting this error:

The Platform Support Image version (server.psi) is lower than the requested schema version

I think I'm following the instructions correctly.  Is there something else I need to do?

Regards,

-John

**Comment by marcus-j-davies** [comment]
Hi @georgeinva2004

Sorry - V4 added some protection with incompatible PSI versions (seems that it works 😄 )
I have now uploaded the latest test PSI (Win x64)

When I come to  releasing V4 - All PSI's will be available

**Comment by georgeinva2004** [comment]
Thank-you!!!!  I'll try it when I get home.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by georgeinva2004** [comment]
That worked!  I did notice that one of the devices (iBlinds controller) now has a Windows Covering command class.  I don't believe that was there before.  It was controlled with the Multilevel Switch command class, but that is no longer available.  Is that something the ZWaveJS server would have introduced?

**Reply by marcus-j-davies** [reply]
Not server - the driver.
The server just pipes the data over.

The latest specification now prohibits the use of Multi Level for blinds or something to that effect.

See : https://github.com/zwave-js/node-zwave-js/pull/5812

_A controlling node MUST NOT interview and provide controlling functionalities for the Multilevel
Switch Command Class for a node (or endpoint) supporting this Command Class, as it is a fully
redundant and less precise application functionality._
